vendor/engines/secure_rooms/app/presenters/secure_rooms/link_collection_extension.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  module LinkCollectionExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      insert_index = tab_methods.index(:admin_products) || -1
```

Comment:
Can we move this one more to the left so it's between "Reservations" and "Billing". Conceptually it fits closer with Orders and Reservations, so I think it should live there.